### PR TITLE
Support SVG for reference images

### PR DESCRIPTION
**Pull request by andybak**
Only thing lacking is export support but I'm not sure what the correct behaviour would be. GLTF supports SVG textures but Blender doesn't seem to allow it. ~~Rasterize the SVGs on export? I think we can punt on this for the moment and document this as a known issue.~~ SVGs are rasterized on export

### CODE REVIEW

**Comment by mikeskydev** [comment]
Cool!

> GLTF supports SVG textures

Source? I've only found info that says it's *not* going to happen officially. https://github.com/KhronosGroup/glTF/issues/549#issuecomment-1542139007

I think it would be fine if we rasterized on export.

File: Packages/manifest.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.6.4",
     "com.unity.ugui": "1.0.0",
+    "com.unity.vectorgraphics": "https://github.com/icosa-gallery/com.unity.vectorgraphics.git#open-brush",
```
Comment (mikeage) [inline Packages/manifest.json:21]: Should this move to the new icosa-mirror org?

File: Packages/packages-lock.json [hunk 2]
```diff
@@ -219,6 +219,25 @@
         "com.unity.modules.imgui": "1.0.0"
       }
     },
+    "com.unity.vectorgraphics": {
+      "version": "https://github.com/icosa-gallery/com.unity.vectorgraphics.git#open-brush",
```
Comment (mikeage) [inline Packages/packages-lock.json:223]: same

**Review by mikeage — COMMENTED** [review]
looks fine, one observation

File: Packages/manifest.json (hunk 1 above)
Comment (mikeskydev) [reply Packages/manifest.json:21]: oh, i didn't see we'd also forked that! yep sure.

Comment (andybak) [reply Packages/manifest.json]: SVGs are now written to png on export.

**Comment by andybak** [comment]
@mikeskydev :

> I think it would be fine if we rasterized on export.

This is now implemented.

**Comment by aaronfranke** [comment]
Is there a desire from Open Brush to support SVG textures in GLTF? I am also mildly interested in having this in Godot Engine. It should be quite trivial to build a spec for this.

The only blocker is justification and motivation. I don't have any particular use case for this myself.

Also, for the name, Khronos discouraged me from using the `EXT_` prefix because there are not multiple vendors interested. However, if both Open Brush and Godot are interested, that would be at least 2 vendors.

**Pull request merged**